src/plays/video-preview/Readme.md

```diff
@@ -0,0 +1,18 @@
+# Video Preview
+
+### User will search a video and when the user hovers over the video thumbnail for 2-5 seconds then user will see a preview of the video
+
+## Play Demographic
+
+- Language: ts
+- Level: Beginner
+
+## Creator Information
+
+- ### **Anshik Gupta** - **[Linkedin](https://www.linkedin.com/in/anshik-gupta-5a99ab192/)**, **[Github](https://github.com/guptaanshik1)**, **[Portfolio](https://me-guptaanshik1.vercel.app/)**
+
+## Implementation Details
+

```

Comment:
Can you please add some implementation details so that others reading this play can know about it.